app/src/modules/settings/routes/data-model/field-detail/store/index.ts

```diff
@@ -95,6 +95,23 @@ export const useFieldDetailStore = defineStore({
 				const relationsStore = useRelationsStore();
 
 				this.field = cloneDeep(fieldsStore.getField(collection, field)!);
+				// re-fetch field meta to get the raw untranslated values
+				// uses callback to avoid blocking the whole startEditing function
```

Comment:
I'm wondering if we should await this anyways. Currently, there's the chance that you'll start editing, and have the field update on you mid-edit. Having to wait for the drawer to open is a slightly better experience IMO

Reply:
> Currently, there's the chance that you'll start editing, and have the field update on you mid-edit

Super agree with this! 👍 

> Having to wait for the drawer to open is a slightly better experience IMO

Implemented this and here's how it looks like with artificial network throttling:

https://user-images.githubusercontent.com/42867097/202666430-f86a3eca-54ab-4f12-a66d-674761d255cb.mp4

but it might be not clear to the user that the drawer is loading before opening as typically drawer opens directly 🤔

Should we either:

- add a loading state to the clicked field as a signal (while also preventing clicking other fields at the same time before the first clicked field loads)?
- or open the drawer first with some form of loading state, then show the drawer fields/tabs afterwards?

Reply:
I think the latter is nicer. Open the drawer immediately, and then use the loading state in the v-form 👍🏻

Reply:
While I was trying to add loading state, I realized adding loading state for _all_ tabs/fields felt a tad excessive, and landed on a middle ground of only adding loading state to the affected fields (field note, interface, display, and validation message):

https://user-images.githubusercontent.com/42867097/203076242-d8a5c1cc-d342-4176-9cfb-f7603f8f43a7.mp4

Thoughts on this? It might be more of a minimal change (so the schema tab doesn't have any loading at all), and technically apart from users with slow network connections, the affect fields will already be loaded before they even view the tabs, so it virtually never shown as loading. (showcased towards the end of the video above, from `0:24` onwards)

Only potential room for improvement might be the skeleton layout for interface and display tab I suppose, if it still feels "not enough loaders".

Reply:
It took a bit of work to even get the loaders to show but i think this partial loading is good-enough for this edge case.